data-pipeline/README.md

```diff
@@ -147,13 +88,12 @@ data-pipeline = { path = ../data-pipeline }
 #### Build and artifact generation
 The building and artifact generation process will depend heavily on the framework selected.
 
-
 ## Expected workflow from teams integrating data-pipeline
 - Create your own language bindings.
 - Hook it in the tracer.
 - Feedback about difficulties about integrating the solution, performance and package size.
 
-## Nice to have duting R&D week
+## Nice to have during R&D week
```

Comment:
should we remove this section since R&D week is over? Or rename the section something along the lines of "Future plans"?

Reply:
Yes, I'll change it.